packages/npm-dependencies-validation/src/utils/packageJsonUtil.ts

```diff
@@ -17,17 +18,18 @@ const pnpmManagerFiles = [
 ];
 const monorepoProps = ["workspaces"];
 
-async function readJsonFile(jsonFilePath: string): Promise<any> {
+async function readJsonFile(jsonFilePath: string): Promise<PackageJson> {
   try {
     const packageJsonContent = await readFile(jsonFilePath, "utf-8");
-    const content: { name: string } = JSON.parse(packageJsonContent);
+    const content: PackageJson = JSON.parse(packageJsonContent);
     return content;
   } catch (error) {
-    return {};
+    // TODO: should we return null/undefined ???
+    return {} as PackageJson;
   }
 }
 
-async function isPathExist(absPath: string): Promise<boolean> {
+export async function isPathExist(absPath: string): Promise<boolean> {
```

Comment:
I think I already commented about the usage of `access` to check if a file exists.
Not sure it is the most suitable way (vs `stat`) also why would we care if there is a **WRITE** permission to the file?
we only ever **READ** the file in these flows.

Reply:
this is the recommended way for this check.
I removed the WRITE permission check for now